ERROR - 2022-10-09 09:05:26 --> Error connecting to the database: ErrorException: pg_connect(): Unable to connect to PostgreSQL server: connection to server at &quot;localhost&quot; (127.0.0.1), port 5432 failed: FATAL:  database &quot;osce_v2&quot; does not exist in /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/Postgre/Connection.php:69
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_connect(): U...', '/Users/ivan/Cod...', 69, Array)
#1 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/Postgre/Connection.php(69): pg_connect('host=localhost ...')
#2 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseConnection.php(382): CodeIgniter\Database\Postgre\Connection->connect(false)
#3 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#4 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM f...', Array, false)
#5 /Users/ivan/Codeigniter/osce_v2/modules/Setting/Models/SettingModel.php(50): CodeIgniter\Database\BaseBuilder->get()
#6 /Users/ivan/Codeigniter/osce_v2/modules/Station/Controllers/Station.php(30): Modules\Setting\Models\SettingModel->getSettingWhere(Array)
#7 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(907): Modules\Station\Controllers\Station->login()
#8 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Station\Controllers\Station))
#9 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /Users/ivan/Codeigniter/osce_v2/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(43): require_once('/Users/ivan/Cod...')
#12 {main}
CRITICAL - 2022-10-09 09:05:26 --> Unable to connect to the database.
Main connection [Postgre]: pg_connect(): Unable to connect to PostgreSQL server: connection to server at &quot;localhost&quot; (127.0.0.1), port 5432 failed: FATAL:  database &quot;osce_v2&quot; does not exist
in SYSTEMPATH/Database/BaseConnection.php on line 418.
 1 SYSTEMPATH/Database/BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM function_login_station(\'Station 1\', \'F305C129\')', [], false)
 3 ROOTPATH/modules/Setting/Models/SettingModel.php(50): CodeIgniter\Database\BaseBuilder->get()
 4 ROOTPATH/modules/Station/Controllers/Station.php(30): Modules\Setting\Models\SettingModel->getSettingWhere([...])
 5 SYSTEMPATH/CodeIgniter.php(907): Modules\Station\Controllers\Station->login()
 6 SYSTEMPATH/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Station\Controllers\Station))
 7 SYSTEMPATH/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Commands/Server/rewrite.php(43): require_once('FCPATH/index.php')
ERROR - 2022-10-09 09:16:35 --> ErrorException: pg_query(): Query failed: ERROR:  function function_tampil_skenario(integer, integer) does not exist
LINE 2: FROM function_tampil_skenario(6, 1)
             ^
HINT:  No function matches the given name and argument types. You might need to add explicit type casts. in /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/Postgre/Connection.php:140
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', '/Users/ivan/Cod...', 140, Array)
#1 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/Postgre/Connection.php(140): pg_query(Resource id #16, 'SELECT *\nFROM f...')
#2 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseConnection.php(693): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM f...')
#3 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM f...')
#4 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM f...', Array, false)
#5 /Users/ivan/Codeigniter/osce_v2/modules/Penilaian/Models/PenilaianModel.php(13): CodeIgniter\Database\BaseBuilder->get()
#6 /Users/ivan/Codeigniter/osce_v2/modules/Penilaian/Controllers/Penilaian.php(34): Modules\Penilaian\Models\PenilaianModel->getSkenario('6', '1')
#7 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(907): Modules\Penilaian\Controllers\Penilaian->index()
#8 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Penilaian\Controllers\Penilaian))
#9 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /Users/ivan/Codeigniter/osce_v2/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(43): require_once('/Users/ivan/Cod...')
#12 {main}
CRITICAL - 2022-10-09 09:16:35 --> pg_query(): Query failed: ERROR:  function function_tampil_skenario(integer, integer) does not exist
LINE 2: FROM function_tampil_skenario(6, 1)
             ^
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
in SYSTEMPATH/Database/Postgre/Connection.php on line 140.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Query failed: ERROR:  function function_tampil_skenario(integer, integer) does not exist
LINE 2: FROM function_tampil_skenario(6, 1)
             ^
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.', 'SYSTEMPATH/Database/Postgre/Connection.php', 140, [...])
 2 SYSTEMPATH/Database/Postgre/Connection.php(140): pg_query(resource (pgsql link), 'SELECT *
FROM function_tampil_skenario(6, 1)')
 3 SYSTEMPATH/Database/BaseConnection.php(693): CodeIgniter\Database\Postgre\Connection->execute('SELECT *
FROM function_tampil_skenario(6, 1)')
 4 SYSTEMPATH/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM function_tampil_skenario(6, 1)')
 5 SYSTEMPATH/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM function_tampil_skenario(6, 1)', [], false)
 6 ROOTPATH/modules/Penilaian/Models/PenilaianModel.php(13): CodeIgniter\Database\BaseBuilder->get()
 7 ROOTPATH/modules/Penilaian/Controllers/Penilaian.php(34): Modules\Penilaian\Models\PenilaianModel->getSkenario('6', '1')
 8 SYSTEMPATH/CodeIgniter.php(907): Modules\Penilaian\Controllers\Penilaian->index()
 9 SYSTEMPATH/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Penilaian\Controllers\Penilaian))
10 SYSTEMPATH/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Commands/Server/rewrite.php(43): require_once('FCPATH/index.php')
ERROR - 2022-10-09 12:21:47 --> ErrorException: pg_query(): Query failed: ERROR:  relation &quot;penguji&quot; does not exist
LINE 2: FROM &quot;penguji&quot;
             ^ in /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/Postgre/Connection.php:140
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', '/Users/ivan/Cod...', 140, Array)
#1 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/Postgre/Connection.php(140): pg_query(Resource id #16, 'SELECT *\nFROM "...')
#2 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseConnection.php(693): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 /Users/ivan/Codeigniter/osce_v2/modules/Penguji/Models/PengujiModel.php(44): CodeIgniter\Database\BaseBuilder->get()
#6 /Users/ivan/Codeigniter/osce_v2/modules/Penguji/Controllers/Penguji.php(98): Modules\Penguji\Models\PengujiModel->getDosen()
#7 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(907): Modules\Penguji\Controllers\Penguji->getPenguji()
#8 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Penguji\Controllers\Penguji))
#9 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /Users/ivan/Codeigniter/osce_v2/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(43): require_once('/Users/ivan/Cod...')
#12 {main}
CRITICAL - 2022-10-09 12:21:47 --> pg_query(): Query failed: ERROR:  relation &quot;penguji&quot; does not exist
LINE 2: FROM &quot;penguji&quot;
             ^
in SYSTEMPATH/Database/Postgre/Connection.php on line 140.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Query failed: ERROR:  relation &quot;penguji&quot; does not exist
LINE 2: FROM &quot;penguji&quot;
             ^', 'SYSTEMPATH/Database/Postgre/Connection.php', 140, [...])
 2 SYSTEMPATH/Database/Postgre/Connection.php(140): pg_query(resource (pgsql link), 'SELECT *
FROM "penguji"')
 3 SYSTEMPATH/Database/BaseConnection.php(693): CodeIgniter\Database\Postgre\Connection->execute('SELECT *
FROM "penguji"')
 4 SYSTEMPATH/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM "penguji"')
 5 SYSTEMPATH/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM "penguji"', [], false)
 6 ROOTPATH/modules/Penguji/Models/PengujiModel.php(44): CodeIgniter\Database\BaseBuilder->get()
 7 ROOTPATH/modules/Penguji/Controllers/Penguji.php(98): Modules\Penguji\Models\PengujiModel->getDosen()
 8 SYSTEMPATH/CodeIgniter.php(907): Modules\Penguji\Controllers\Penguji->getPenguji()
 9 SYSTEMPATH/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Penguji\Controllers\Penguji))
10 SYSTEMPATH/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Commands/Server/rewrite.php(43): require_once('FCPATH/index.php')
CRITICAL - 2022-10-09 12:21:47 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Format/JSONFormatter.php:41
Stack trace:
#0 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Format/JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/API/ResponseTrait.php(348): CodeIgniter\Format\JSONFormatter->format(Array)
#2 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/API/ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Debug/Exceptions.php(132): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
in SYSTEMPATH/Format/JSONFormatter.php on line 41.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2022-10-09 12:21:48 --> ErrorException: pg_query(): Query failed: ERROR:  relation &quot;penguji&quot; does not exist
LINE 2: FROM &quot;penguji&quot;
             ^ in /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/Postgre/Connection.php:140
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', '/Users/ivan/Cod...', 140, Array)
#1 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/Postgre/Connection.php(140): pg_query(Resource id #16, 'SELECT *\nFROM "...')
#2 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseConnection.php(693): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 /Users/ivan/Codeigniter/osce_v2/modules/Penguji/Models/PengujiModel.php(44): CodeIgniter\Database\BaseBuilder->get()
#6 /Users/ivan/Codeigniter/osce_v2/modules/Penguji/Controllers/Penguji.php(98): Modules\Penguji\Models\PengujiModel->getDosen()
#7 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(907): Modules\Penguji\Controllers\Penguji->getPenguji()
#8 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Penguji\Controllers\Penguji))
#9 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /Users/ivan/Codeigniter/osce_v2/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(43): require_once('/Users/ivan/Cod...')
#12 {main}
CRITICAL - 2022-10-09 12:21:48 --> pg_query(): Query failed: ERROR:  relation &quot;penguji&quot; does not exist
LINE 2: FROM &quot;penguji&quot;
             ^
in SYSTEMPATH/Database/Postgre/Connection.php on line 140.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Query failed: ERROR:  relation &quot;penguji&quot; does not exist
LINE 2: FROM &quot;penguji&quot;
             ^', 'SYSTEMPATH/Database/Postgre/Connection.php', 140, [...])
 2 SYSTEMPATH/Database/Postgre/Connection.php(140): pg_query(resource (pgsql link), 'SELECT *
FROM "penguji"')
 3 SYSTEMPATH/Database/BaseConnection.php(693): CodeIgniter\Database\Postgre\Connection->execute('SELECT *
FROM "penguji"')
 4 SYSTEMPATH/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM "penguji"')
 5 SYSTEMPATH/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM "penguji"', [], false)
 6 ROOTPATH/modules/Penguji/Models/PengujiModel.php(44): CodeIgniter\Database\BaseBuilder->get()
 7 ROOTPATH/modules/Penguji/Controllers/Penguji.php(98): Modules\Penguji\Models\PengujiModel->getDosen()
 8 SYSTEMPATH/CodeIgniter.php(907): Modules\Penguji\Controllers\Penguji->getPenguji()
 9 SYSTEMPATH/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Penguji\Controllers\Penguji))
10 SYSTEMPATH/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Commands/Server/rewrite.php(43): require_once('FCPATH/index.php')
CRITICAL - 2022-10-09 12:21:49 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Format/JSONFormatter.php:41
Stack trace:
#0 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Format/JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/API/ResponseTrait.php(348): CodeIgniter\Format\JSONFormatter->format(Array)
#2 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/API/ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Debug/Exceptions.php(132): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
in SYSTEMPATH/Format/JSONFormatter.php on line 41.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2022-10-09 12:22:13 --> ErrorException: pg_query(): Query failed: ERROR:  relation &quot;penguji&quot; does not exist
LINE 2: FROM &quot;penguji&quot;
             ^ in /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/Postgre/Connection.php:140
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', '/Users/ivan/Cod...', 140, Array)
#1 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/Postgre/Connection.php(140): pg_query(Resource id #16, 'SELECT *\nFROM "...')
#2 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseConnection.php(693): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 /Users/ivan/Codeigniter/osce_v2/modules/Penguji/Models/PengujiModel.php(44): CodeIgniter\Database\BaseBuilder->get()
#6 /Users/ivan/Codeigniter/osce_v2/modules/Penguji/Controllers/Penguji.php(98): Modules\Penguji\Models\PengujiModel->getDosen()
#7 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(907): Modules\Penguji\Controllers\Penguji->getPenguji()
#8 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Penguji\Controllers\Penguji))
#9 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /Users/ivan/Codeigniter/osce_v2/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(43): require_once('/Users/ivan/Cod...')
#12 {main}
CRITICAL - 2022-10-09 12:22:13 --> pg_query(): Query failed: ERROR:  relation &quot;penguji&quot; does not exist
LINE 2: FROM &quot;penguji&quot;
             ^
in SYSTEMPATH/Database/Postgre/Connection.php on line 140.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Query failed: ERROR:  relation &quot;penguji&quot; does not exist
LINE 2: FROM &quot;penguji&quot;
             ^', 'SYSTEMPATH/Database/Postgre/Connection.php', 140, [...])
 2 SYSTEMPATH/Database/Postgre/Connection.php(140): pg_query(resource (pgsql link), 'SELECT *
FROM "penguji"')
 3 SYSTEMPATH/Database/BaseConnection.php(693): CodeIgniter\Database\Postgre\Connection->execute('SELECT *
FROM "penguji"')
 4 SYSTEMPATH/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM "penguji"')
 5 SYSTEMPATH/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM "penguji"', [], false)
 6 ROOTPATH/modules/Penguji/Models/PengujiModel.php(44): CodeIgniter\Database\BaseBuilder->get()
 7 ROOTPATH/modules/Penguji/Controllers/Penguji.php(98): Modules\Penguji\Models\PengujiModel->getDosen()
 8 SYSTEMPATH/CodeIgniter.php(907): Modules\Penguji\Controllers\Penguji->getPenguji()
 9 SYSTEMPATH/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Penguji\Controllers\Penguji))
10 SYSTEMPATH/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Commands/Server/rewrite.php(43): require_once('FCPATH/index.php')
CRITICAL - 2022-10-09 12:22:13 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Format/JSONFormatter.php:41
Stack trace:
#0 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Format/JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/API/ResponseTrait.php(348): CodeIgniter\Format\JSONFormatter->format(Array)
#2 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/API/ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Debug/Exceptions.php(132): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
in SYSTEMPATH/Format/JSONFormatter.php on line 41.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2022-10-09 12:22:14 --> ErrorException: pg_query(): Query failed: ERROR:  relation &quot;penguji&quot; does not exist
LINE 2: FROM &quot;penguji&quot;
             ^ in /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/Postgre/Connection.php:140
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', '/Users/ivan/Cod...', 140, Array)
#1 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/Postgre/Connection.php(140): pg_query(Resource id #16, 'SELECT *\nFROM "...')
#2 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseConnection.php(693): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 /Users/ivan/Codeigniter/osce_v2/modules/Penguji/Models/PengujiModel.php(44): CodeIgniter\Database\BaseBuilder->get()
#6 /Users/ivan/Codeigniter/osce_v2/modules/Penguji/Controllers/Penguji.php(98): Modules\Penguji\Models\PengujiModel->getDosen()
#7 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(907): Modules\Penguji\Controllers\Penguji->getPenguji()
#8 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Penguji\Controllers\Penguji))
#9 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /Users/ivan/Codeigniter/osce_v2/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(43): require_once('/Users/ivan/Cod...')
#12 {main}
CRITICAL - 2022-10-09 12:22:14 --> pg_query(): Query failed: ERROR:  relation &quot;penguji&quot; does not exist
LINE 2: FROM &quot;penguji&quot;
             ^
in SYSTEMPATH/Database/Postgre/Connection.php on line 140.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Query failed: ERROR:  relation &quot;penguji&quot; does not exist
LINE 2: FROM &quot;penguji&quot;
             ^', 'SYSTEMPATH/Database/Postgre/Connection.php', 140, [...])
 2 SYSTEMPATH/Database/Postgre/Connection.php(140): pg_query(resource (pgsql link), 'SELECT *
FROM "penguji"')
 3 SYSTEMPATH/Database/BaseConnection.php(693): CodeIgniter\Database\Postgre\Connection->execute('SELECT *
FROM "penguji"')
 4 SYSTEMPATH/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM "penguji"')
 5 SYSTEMPATH/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM "penguji"', [], false)
 6 ROOTPATH/modules/Penguji/Models/PengujiModel.php(44): CodeIgniter\Database\BaseBuilder->get()
 7 ROOTPATH/modules/Penguji/Controllers/Penguji.php(98): Modules\Penguji\Models\PengujiModel->getDosen()
 8 SYSTEMPATH/CodeIgniter.php(907): Modules\Penguji\Controllers\Penguji->getPenguji()
 9 SYSTEMPATH/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Penguji\Controllers\Penguji))
10 SYSTEMPATH/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Commands/Server/rewrite.php(43): require_once('FCPATH/index.php')
CRITICAL - 2022-10-09 12:22:14 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Format/JSONFormatter.php:41
Stack trace:
#0 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Format/JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/API/ResponseTrait.php(348): CodeIgniter\Format\JSONFormatter->format(Array)
#2 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/API/ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Debug/Exceptions.php(132): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
in SYSTEMPATH/Format/JSONFormatter.php on line 41.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2022-10-09 12:22:15 --> ErrorException: pg_query(): Query failed: ERROR:  relation &quot;penguji&quot; does not exist
LINE 2: FROM &quot;penguji&quot;
             ^ in /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/Postgre/Connection.php:140
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', '/Users/ivan/Cod...', 140, Array)
#1 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/Postgre/Connection.php(140): pg_query(Resource id #16, 'SELECT *\nFROM "...')
#2 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseConnection.php(693): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 /Users/ivan/Codeigniter/osce_v2/modules/Penguji/Models/PengujiModel.php(44): CodeIgniter\Database\BaseBuilder->get()
#6 /Users/ivan/Codeigniter/osce_v2/modules/Penguji/Controllers/Penguji.php(98): Modules\Penguji\Models\PengujiModel->getDosen()
#7 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(907): Modules\Penguji\Controllers\Penguji->getPenguji()
#8 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Penguji\Controllers\Penguji))
#9 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /Users/ivan/Codeigniter/osce_v2/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(43): require_once('/Users/ivan/Cod...')
#12 {main}
CRITICAL - 2022-10-09 12:22:15 --> pg_query(): Query failed: ERROR:  relation &quot;penguji&quot; does not exist
LINE 2: FROM &quot;penguji&quot;
             ^
in SYSTEMPATH/Database/Postgre/Connection.php on line 140.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Query failed: ERROR:  relation &quot;penguji&quot; does not exist
LINE 2: FROM &quot;penguji&quot;
             ^', 'SYSTEMPATH/Database/Postgre/Connection.php', 140, [...])
 2 SYSTEMPATH/Database/Postgre/Connection.php(140): pg_query(resource (pgsql link), 'SELECT *
FROM "penguji"')
 3 SYSTEMPATH/Database/BaseConnection.php(693): CodeIgniter\Database\Postgre\Connection->execute('SELECT *
FROM "penguji"')
 4 SYSTEMPATH/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM "penguji"')
 5 SYSTEMPATH/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM "penguji"', [], false)
 6 ROOTPATH/modules/Penguji/Models/PengujiModel.php(44): CodeIgniter\Database\BaseBuilder->get()
 7 ROOTPATH/modules/Penguji/Controllers/Penguji.php(98): Modules\Penguji\Models\PengujiModel->getDosen()
 8 SYSTEMPATH/CodeIgniter.php(907): Modules\Penguji\Controllers\Penguji->getPenguji()
 9 SYSTEMPATH/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Penguji\Controllers\Penguji))
10 SYSTEMPATH/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Commands/Server/rewrite.php(43): require_once('FCPATH/index.php')
CRITICAL - 2022-10-09 12:22:15 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Format/JSONFormatter.php:41
Stack trace:
#0 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Format/JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/API/ResponseTrait.php(348): CodeIgniter\Format\JSONFormatter->format(Array)
#2 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/API/ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Debug/Exceptions.php(132): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
in SYSTEMPATH/Format/JSONFormatter.php on line 41.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2022-10-09 12:22:21 --> ErrorException: pg_query(): Query failed: ERROR:  relation &quot;penguji&quot; does not exist
LINE 2: FROM &quot;penguji&quot;
             ^ in /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/Postgre/Connection.php:140
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', '/Users/ivan/Cod...', 140, Array)
#1 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/Postgre/Connection.php(140): pg_query(Resource id #16, 'SELECT *\nFROM "...')
#2 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseConnection.php(693): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 /Users/ivan/Codeigniter/osce_v2/modules/Penguji/Models/PengujiModel.php(44): CodeIgniter\Database\BaseBuilder->get()
#6 /Users/ivan/Codeigniter/osce_v2/modules/Penguji/Controllers/Penguji.php(98): Modules\Penguji\Models\PengujiModel->getDosen()
#7 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(907): Modules\Penguji\Controllers\Penguji->getPenguji()
#8 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Penguji\Controllers\Penguji))
#9 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /Users/ivan/Codeigniter/osce_v2/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(43): require_once('/Users/ivan/Cod...')
#12 {main}
CRITICAL - 2022-10-09 12:22:21 --> pg_query(): Query failed: ERROR:  relation &quot;penguji&quot; does not exist
LINE 2: FROM &quot;penguji&quot;
             ^
in SYSTEMPATH/Database/Postgre/Connection.php on line 140.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Query failed: ERROR:  relation &quot;penguji&quot; does not exist
LINE 2: FROM &quot;penguji&quot;
             ^', 'SYSTEMPATH/Database/Postgre/Connection.php', 140, [...])
 2 SYSTEMPATH/Database/Postgre/Connection.php(140): pg_query(resource (pgsql link), 'SELECT *
FROM "penguji"')
 3 SYSTEMPATH/Database/BaseConnection.php(693): CodeIgniter\Database\Postgre\Connection->execute('SELECT *
FROM "penguji"')
 4 SYSTEMPATH/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM "penguji"')
 5 SYSTEMPATH/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM "penguji"', [], false)
 6 ROOTPATH/modules/Penguji/Models/PengujiModel.php(44): CodeIgniter\Database\BaseBuilder->get()
 7 ROOTPATH/modules/Penguji/Controllers/Penguji.php(98): Modules\Penguji\Models\PengujiModel->getDosen()
 8 SYSTEMPATH/CodeIgniter.php(907): Modules\Penguji\Controllers\Penguji->getPenguji()
 9 SYSTEMPATH/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(Modules\Penguji\Controllers\Penguji))
10 SYSTEMPATH/CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Commands/Server/rewrite.php(43): require_once('FCPATH/index.php')
CRITICAL - 2022-10-09 12:22:21 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Format/JSONFormatter.php:41
Stack trace:
#0 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Format/JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/API/ResponseTrait.php(348): CodeIgniter\Format\JSONFormatter->format(Array)
#2 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/API/ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 /Users/ivan/Codeigniter/osce_v2/vendor/codeigniter4/framework/system/Debug/Exceptions.php(132): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
in SYSTEMPATH/Format/JSONFormatter.php on line 41.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
